src/subscript/welltest_dpds/welltest_dpds.py

```diff
@@ -452,15 +502,34 @@ def main():
         dp, dspt, super_time, wbhp, bu_start_ind, bu_end_ind
     )
 
+    if genobs_resultf:
+        dpdspt_w1_gendata = genobs_vec(genobs_resultf, dpdspt_weighted_lag1, cum_time)
+        dpdspt_w2_gendata = genobs_vec(genobs_resultf, dpdspt_weighted_lag2, cum_time)
+        wbhp_gendata = genobs_vec(
+            genobs_resultf, wbhp[bu_start_ind + 1 : bu_end_ind + 1], cum_time
+        )
+
+        to_csv(
+            outdir + "dpdspt_lag1_genobs" + outf_suffix + ".csv",
+            [dpdspt_w1_gendata],
+        )
+        to_csv(
+            outdir + "dpdspt_lag2_genobs" + outf_suffix + ".csv",
+            [dpdspt_w2_gendata],
+        )
+        to_csv(
+            outdir + "wbhp_genobs" + outf_suffix + ".csv",
+            [wbhp_gendata],
+        )
     to_csv(
         outdir + "dpdspt_lag1" + outf_suffix + ".csv",
         [cum_time, dpdspt_weighted_lag1],
-        ["HOURS", "dpd(supt)_w"],
+        ["Hours", "dpd(supt)_w"],
```

Comment:
Back to non-standard?

Reply:
User request again. Is there a clearly defined standard?